tasks/installer.py

```diff
@@ -14,7 +13,7 @@
 from tasks.go import deps
 from tasks.libs.common.utils import REPO_PATH, bin_name, get_build_flags, get_version, load_release_versions, timed
 
-BIN_PATH = os.path.join(".", "bin", "updater")
+BIN_PATH = os.path.join(".", "bin", "installer")
```

Comment:
won't this break with systemd unit path still targetting bin/updater?

Reply:
Most likely but since this this renaming is handled in your PR I assumed it would be OK.

I can also push these changes in your branches to have all renaming in a single PR/commit, which might ease up potential reverts

Reply:
I did not rename binary paths though, didn't get the time to prepare it yet. I'll do it on top of your PR

Reply:
Reverted the binary name part